### PR TITLE
Create miz_vivi.txt

### PR DESCRIPTION
**Pull request by CHARLESojini**

### CODE REVIEW

**Comment by obinnaaliogor** [comment]
Please review team @CHARLESojini @Engrlyndi @Jazzmangit @mamaputt @obinnaaliogor

**Comment by CHARLESojini** [comment]
Good to GO

File: miz_vivi.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+hair poducts on sales
+50% discount
+thanks for comming
```
Comment (Engrlyndi) [inline miz_vivi.txt:3]: kindly offer 70%

**Comment by obinnaaliogor** [comment]
Good job team, for the confirmation @mamaputt @Engrlyndi @Jazzmangit  @obinnaaliogor

**Review by mamaputt — COMMENTED** [review]
review accepted,

**Comment by CHARLESojini** [comment]
GOOD

**Pull request merged**